qa/tasks/cephfs/test_quota.py

```diff
@@ -104,3 +104,69 @@ def test_remote_update_write(self):
         with self.assertRaises(CommandFailedError):
             self.mount_b.write_n_mb("subdir_data/file", 40)
 
+    def test_human_readable_quota_values(self):
+        self.mount_a.run_shell(["mkdir", "subdir"])
+
+        self.assertEqual(self.mount_a.getfattr("./subdir",
+                                               "ceph.quota.max_bytes"), None)
+
+        # kibibyte
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "10K")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "10240")
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "100Ki")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "102400")
+
+        # mebibyte
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "10M")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "10485760")
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "100Mi")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "104857600")
+
+        # gibibyte
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "2G")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "2147483648")
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "4Gi")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "4294967296")
+
+        # tebibyte
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "1T")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "1099511627776")
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "2Ti")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "2199023255552")
+
+        # invalid arguments
```

Comment:
nit: For these cases a separate method would be nicer. It makes recognizing and understanding test failures easier. It can be named something like `test_human_readable_quota_invalid_values`.

Reply:
done